src/sentry/tasks/store.py

```diff
@@ -57,12 +58,16 @@ def _do_preprocess_event(cache_key, data, start_time, event_id, process_event):
         error_logger.error('preprocess.failed.empty', extra={'cache_key': cache_key})
         return
 
-    project = data['project']
+    project_id = data['project']
     Raven.tags_context({
-        'project': project,
+        'project': project_id,
     })
 
     if should_process(data):
+        # save another version of data for some projects to generate
+        # preprocessing hash that won't be modified by pipeline
+        hash_cache.set(get_raw_cache_key(project_id, data['event_id']), data)
```

Comment:
It's not going to affect anything right now but I figured it's worth noting that this data is not /literally/ the same data that will be used for filtering, since this cache set happens after scrubbing. I don't anticipate it causing any problems with the current implementation but could possibly lead to strange behavior in the future if the hash components or scrubbers are modified and result the hash components including any data that was scrubbed (intentionally or not.) I'm not sure of a more robust solution here without giving it more thought.

Reply:
ah good point. hmmmmm 🤔

Reply:
i can at the very least add a comment in preprocess_hashes.py in my other PR noting that so if we do wind up changing stuff, we at least don't forget about it